gradio/routes.py

```diff
@@ -1185,7 +1186,7 @@ async def analytics_login(request: fastapi.Request):
 
         @app.get("/monitoring/{key}")
         async def analytics_dashboard(key: str):
-            if key == app.analytics_key:
```

Comment:
can we use `route_utils.compare_passwords_securely` for consistency?

Reply:
good call thanks!